internal/watch/watch.go

```diff
@@ -288,3 +289,95 @@ func renderLogs(ctx context.Context, alloc *api.AllocationStatus) {
 		)
 	}
 }
+
+func MachinesChecks(ctx context.Context) error {
+	var (
+		io          = iostreams.FromContext(ctx)
+		colorize    = io.ColorScheme()
+		flapsClient = flaps.FromContext(ctx)
+	)
+	var errCount int
+
+	ctx, cancel := context.WithTimeout(ctx, 300*time.Second)
+	defer cancel()
+
+	fmt.Fprintln(io.Out)
+	fmt.Fprintln(io.Out, colorize.Green("==> "+"Monitoring health checks"))
+
+	for {
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		case <-time.After(100 * time.Millisecond):
```

Comment:
My understanding is that this can lead to a memory leak: https://www.arangodb.com/2020/09/a-story-of-a-memory-leak-in-go-how-to-properly-use-time-after/

Also, we should extend this interval a bit. 100ms is pretty aggressive for a loop that hits an external API.  1-2 seconds should be ok.